src/agent/TransportService.ts

```diff
@@ -0,0 +1,41 @@
+import { Socket } from 'socket.io';
```

Comment:
I've mentioned this before, but AFAIK socket.io uses non-standard communication protocols. This means this won't work with ACA-Py as a mediator.

I have objections to merging this with socket.io instead of a general WS server. If you can make it such that socket.io can communicate with non-socket.io agents that's also fine

Reply:
As @TimoGlastra suggests, Javascript WebSocket works with ACA-py. I have tested this in my own code and works. https://developer.mozilla.org/en-US/docs/Web/API/WebSocket